src/components/Drawer.jsx

```diff
@@ -12,6 +12,10 @@ const DrawerContent = styled.div`
 	font-size: 14px;
 `
 
+const MUIDrawerStyled = styled(MUIDrawer)`
+	z-index: 1000;

```

Comment:
Let's make this a bit larger than the z-index of the top component because then the drawers gets under it and prevents users from saving, closing, etc. 
![image](https://user-images.githubusercontent.com/23409284/169414615-f4415985-4b38-4593-899f-b0face9eebc3.png)